store/storetest/team_store.go

```diff
@@ -1433,9 +1453,9 @@ func testTeamStoreMigrateTeamMembers(t *testing.T, ss store.Store) {
 		ExplicitRoles: "something_else",
 	}
 
-	tm1 = (<-ss.Team().SaveMember(tm1, -1)).Data.(*model.TeamMember)
-	tm2 = (<-ss.Team().SaveMember(tm2, -1)).Data.(*model.TeamMember)
-	tm3 = (<-ss.Team().SaveMember(tm3, -1)).Data.(*model.TeamMember)
+	tm1, _ = ss.Team().SaveMember(tm1, -1)
```

Comment:
not in the previous code, but we can add the `require.Nil(t, err)` here too (and in the next 2 lines)

Reply:
alright